certs/api/logging.go

```diff
@@ -90,3 +90,42 @@ func (lm *loggingMiddleware) RevokeCert(ctx context.Context, token, thingID stri
 
 	return lm.svc.RevokeCert(ctx, token, thingID)
 }
+
+func (lm *loggingMiddleware) RenewCerts(ctx context.Context, bsUpdateRenewCert bool) (err error) {
```

Comment:
Please align the log message with the rest of the service: `Method <method name> <other useful info> took <duration> to complete <errors if any>`.

Reply:
The method returns only error 
The error contains all the details,